util/Makefile

```
@@ -4,17 +4,24 @@ TARGET = zenfs
 
 CC ?= gcc
 CXX ?= g++
+OBJCOPY ?= objcopy
 
 CXXFLAGS = $(shell pkg-config --cflags rocksdb)
 LIBS = $(shell pkg-config --static --libs rocksdb)
 
 CXXFLAGS +=  $(EXTRA_CXXFLAGS)
 LDFLAGS +=  $(EXTRA_LDFLAGS)
 
-all: $(TARGET)
+all: $(TARGET) generate_dbg
+
+generate_dbg:
+	@$(OBJCOPY) --only-keep-debug $(TARGET) $(TARGET).dbg
+	@$(OBJCOPY) --strip-debug $(TARGET)
```

```suggestion
$(TARGET).dbg:
	@$(OBJCOPY) --only-keep-debug $(TARGET) $(TARGET).dbg
	@strip $(TARGET)
```

I think strip removes more stuff than objcpy --strip-debug. Could be that objcopy can do the same as strip with other options.

---

I think using one command (objcopy or strip) in both places is better,  the strip man page also uses objcopy --strip-debug. Post strip, the difference in size is not huge between strip/objcopy.  I feel nothing wrong with using 
"objcopy --strip-debug"

---

'objcopy --strip-all' does the same thing as strip binary does, changed it.